packages/validation/src/MonokleValidator.ts

```diff
@@ -172,16 +103,17 @@ export class MonokleValidator implements Validator {
    * Eagerly load and configure the validation plugins.
    *
    * @param config - the new configuration of the validator.
+   * @param suppressions - a list with suppression requests.
    */
   async preload(config?: Config, suppressions?: Suppression[]): Promise<void> {
-    this._config = config;
-    this._suppressions = suppressions || [];
+    this._config = config ?? {};
+    this._suppressions = suppressions ?? [];
```

Comment:
I would go for `||` here because it covers more cases (including `??`)

Reply:
Not fully agreeing here :D While it covers more cases, `??` has been introduced because `||` tends to introduce subtle bugs because these additional cases are often not the ones you want.

Reply:
I'm aware of those case, but none of them apply here. :P